src/ui/modals/confirm_queue_clear.rs

```diff
@@ -120,7 +115,47 @@ impl Modal for ConfirmQueueClearModal<'_> {
                 }
                 _ => {}
             }
-        };
+        } else if let Some(action) = key.as_global_action(context) {
```

Comment:
@mierak This is this what you meant, right?

Reply:
Yup. `stop_propagation` should not be needed now since the event gets consumed automatically by `as_global_action`. You can also use the default branch to not have to list all the unused binds.

Reply:
ah ok that's good to hear :+1: